src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  ContentRecord,
+  CubeName,
+  DashboardState,
+  DataModel,
+  deserializeDashboardState,
+  MdxSelect,
+  serializeDashboardState,
```

Comment:
This should be imported from 5.1

Reply:
I meant `serializeDashboardState` only, not everything.
What matters the most is that it's `deserialize` from `5.0` and `serialize` from `5.1`, since we respectively parse a 5.0 content and create a new 5.1 content.